pkgs/top-level/all-packages.nix

```diff
@@ -5581,6 +5581,21 @@ with pkgs;
 
   gcc_multi = wrapCCMulti gcc;
 
+  llvm-multi = llvmPkgs:
```

Comment:
This does not look like a good name, since it seems to be about clang, not llvm in general. Maybe `wrapClangMulti`?